apps/website/foundation/charts/README.md

```diff
@@ -0,0 +1,253 @@
+## Process
+
+:::
+
+::: component-section-level-two-title
+
+### Selecting Colors
+
+:::
+
+::: component-section-level-two
+
+Examine your data - are the relationships between data points clarified by one of the
+definitions above? (Categorical, sequential, diverging)? Selecting colors that reveal the
+attributes which differentiate between data points is essential - rhi is not art, choosing
+pleasing colors is desirable but must be done in the context of the relationships otherwise
+you risk creating an ambiguous or miselading chart.
+
+We recommend using one of the sets provided.
+
+:::
+
+::: component-section-level-two-title
+
+### Differentiate between Samples
+
+:::
+
+::: component-section-level-two
+
+Not all users can identify the differences between samples that may be apparent to you.
+A11y compliance requires a contrast ratio of 3:1 between adjacent colors - consider using
+a white or black border between adjacent colors to meet this requirement.
+
+:::
+
+::: component-section-level-two-title
+
+### Use Texture
+
+:::
+
+::: component-section-level-two
+
+Some users will not be able to differentiate between different colors in situations where an obvious
+border doesn't exist for instance a line chart. Adding textures to the visualizations can make
+the different trend lines clearly distinct from one another.
+
+:::
+
+::: component-section-level-two-title
+
+### Legends vs Labels
+
+:::
+
+::: component-section-level-two
+
+Whenever possible label data directly - legends are not useful to all users and violate the
+preference for proximity in UX.
+
+:::
+
+::: component-section-level-one-title
+
+## A11Y
+
+:::
+
+::: component-section-level-one
+
+Accessibility is a very important and challenging aspect of data visualization work.
+There are several techniques one may employ to improve the accessibility of charts
+but ultimately the challenge is so formidable that offering an alternate view such as a table,
+is often necessary. Assuming you are dedicated to making your charts as accessible as possible,
+here are few helpful tips:
+
+<br/>
+
+1. Color accessibility derives from adjacent contrast. Two colors may have sufficient contrast (3.1:1) when
+   measured against the background, but they may also be below the threshold relative to one another.
+   Using high contrast separations between adjacent colors (such as black or white borders) will often solve this.
+2. Textures provide a method of differentiating one area from the next which is not dependent on color.
+   Consider if using texture for your charts is appropriate. Clarity provides the following textures - remember that
+   the texture itself must be of sufficient contrast (3.1:1) against its background in order to be accessible.
+3. Direct labeling. Accessibility is improved by maintaining close proximity between labels and the item they refer to.
+   Legends can be less useful particularly in magnification use. In the example below the correspondence between blue in
```

Comment:
Do you have screenshots for those left/right examples?

Reply:
Jeremie, live demo is available here: https://6025b50934b3380a109440e6--vmware-clarity.netlify.app/foundation/charts/

Reply:
Thanks @amalitsky, I have indeed been using that preview, it's very useful and helpful!
However, the examples you mention there seem missing:

<img width="1089" alt="Screen Shot 2021-02-24 at 10 09 39 PM" src="https://user-images.githubusercontent.com/113730/109052857-198b2780-76aa-11eb-81b1-2dd2b6136305.png">

Reply:
Got the question now, thanks.

There was no illustration originally planned for the highlighted section.

That said, content is not in it's final form at this point. Meaning both - text as well as illustrations might change based on the review and content provided by the design team.